src/h5web/metadata-viewer/MetadataViewer.test.tsx

```diff
@@ -31,7 +31,7 @@ describe('MetadataViewer', () => {
     fireEvent.click(await screen.findByRole('tab', { name: 'Inspect' }));
     await selectExplorerNode('entities');
 
-    const column = screen.getByRole('columnheader', { name: /Group/u });
+    const column = await screen.findByRole('columnheader', { name: /Group/u });
```

Comment:
The entity's metadata is not longer available to the `MetadataViewer` instantly.